pkg/transferer/transferer.go

```diff
@@ -200,7 +202,106 @@ func (t Transferer) writeChunksToFile(ctx context.Context, meta dump.Meta, chunk
 	}
 }
 
-func (t Transferer) Export(ctx context.Context, lc LoadStatusGetter, meta dump.Meta, pool ChunkPool) error {
+type DumpFile struct {
+	header *tar.Header
+	data   []byte
+}
+
+func ReadDump(file *os.File) ([]DumpFile, error) {
+	gzr, err := gzip.NewReader(file)
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to create gzip reader")
+	}
+
+	tr := tar.NewReader(gzr)
+
+	files := make([]DumpFile, 0)
+
+	// Reading existing files into buffer
+	for {
+		header, err := tr.Next()
+		if err == nil {
+			data := make([]byte, header.Size)
+			_, err = tr.Read(data)
+			if err == nil || err == io.EOF {
+				files = append(files, DumpFile{header, data})
+			}
+		} else {
+			break
+		}
+	}
+
+	err = gzr.Close()
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to close gzip reader")
+	}
+
+	return files, nil
+}
+
+func (t Transferer) WriteLog(byteLog []byte) error {
```

Comment:
What will happen if we use `--stdout` option?

We should write logs in `writeChunksToFile` function after writing meta file